example_community_patch_settings.toml

```diff
@@ -224,8 +224,14 @@ select_ship8 = '8'
 # Show alliance screen
 show_alliance = '\'
 
+# Show alliance armada screen
+show_alliance_armada = 'CTRL-\'
+
+# Show alliance help screen
+show_alliance_help = 'SHIFT-\'
+
```

Comment:
Technically these are experimental options that aren't in a full release yet.  But does no harm aside from people wondering why they don't work.